Kotlin/app/src/main/java/com/taboola/kotlin/examples/screens/web/TaboolaWebWrapperViewModel.kt

```diff
@@ -32,7 +32,26 @@ class TaboolaWebWrapperViewModel: ViewModel() {
      * This method is just a quick way to load an example page simulating customer layout.
      * Inside the asset file there are Javascript tags Taboola targets with its content.
      */
-    fun loadWebViewContent(webView: WebView, properties: PlacementInfo.Properties, context: Context) {
+    fun loadWebViewContentWidget(webView: WebView, properties: PlacementInfo.WidgetProperties, context: Context) {
+        val BASE_URL = "https://example.com"
+        val HTML_CONTENT_FILE_TITLE = "sampleContentPage.html"
+        var htmlContent: String? = null
+        try {
```

Comment:
@voltfmanSlava  - Could use .apply to combine all the assignments to htmlConent.

Reply:
Done

Reply:
@tomer-br-taboola doesn't work

Reply:
@voltfmanSlava  - What exactly doesn't work? Can you elaborate?

Reply:
@tomer-br-taboola when you get to "webView.loadDataWithBaseURL(BASE_URL, htmlContent, "text/html", "UTF-8", "")" line htmlContent doesn't contain swapped data it still contains placeHolders

Reply:
@tomer-br-taboola first option isn't compiling. found another option, is it ok?

Reply:
@voltfmanSlava - Yes